File: moforum/src/main/java/com/moforum/entity/Reply.java
package com.moforum.entity;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class Reply {
    private Long id;
    private Long postId;
    private Long userId;
    private String content;
    private LocalDateTime createTime;
    /** 联表查询：作者昵称 */
    private String authorName;
}

File: moforum-web/src/api/reply.ts
import { http } from './http'

export type ReplyRow = {
  id: number
  postId: number
  userId: number
  content: string
  createTime?: string
  authorName?: string
  avatarUrl?: string
}

export type CreateReplyRes = { success: boolean; message: string }

export function listReplies(postId: number) {
  return http.get<ReplyRow[]>('/reply/list', { params: { postId } })
}

export function createReply(payload: {
  postId: number
  content: string
}) {
  return http.post<CreateReplyRes>('/reply/create', payload)
}
